### PR TITLE
More ship names, hails, and commodities

### PR DESCRIPTION
**Pull request by Pointedstick**

### CODE REVIEW

File: data/hails.txt [hunk 1]
```diff
@@ -479,6 +490,8 @@ phrase "friendly syndicate"
 		"! Were you aware of our Plutonium recycling program?"
 		"! Did you know that the average ship contains 39% Syndicated Systems components by weight?"
 		"! If you're looking for affordable radioisotopes, you've come to the right place!"
+		"! Apply to become a Syndicated captain today!"
+		"! We welcome any and all application to join the Syndicated systems team!"
```
Comment (endless-sky) [inline data/hails.txt]: "applications" should be plural.

**Comment by endless-sky** [comment]
I marked one typo, but other than that this looks really good!

**Comment by Pointedstick** [comment]
Aargh, thanks!

**Comment by Pointedstick** [comment]
Fixed that, and also added some more electronics.

**Comment by johnca** [comment]
Bazinje - heh

**Pull request merged**